examples/cra-kitchen-sink/src/stories/index.js

```diff
@@ -78,7 +78,7 @@ storiesOf('Button', module)
           {intro}
         </p>
         <p>
-          My birthday is: {new Date(birthday).toLocaleDateString()}
+          My birthday is: {new Date(birthday).toDateString()}
```

Comment:
This is still locale dependant unfortunately.

I think the solution would be to keep using `toLocaleDateString` but pass in a locale and a format option.

```
var options = { weekday: 'short', year: 'numeric', month: 'short', day: 'numeric' };
date.toLocaleDateString('en-US', options);
```

or something like this.